### PR TITLE
Fix darwin load file type

### PR DESCRIPTION
**Pull request by ccqpein**
fix the issue #49

### CODE REVIEW

File: libraries.lisp [hunk 1]
```diff
@@ -11,7 +11,8 @@
 (eval-when (:compile-toplevel :load-toplevel :execute)
   (cffi:define-foreign-library grpc-client-wrapper
 	;; Load lib on mac
-	(:darwin (:default "libgrpc"))
+	(:darwin #.(namestring
+				(asdf:system-relative-pathname "grpc" "grpc.so")))
```
Comment (Slids) [inline libraries.lisp:15]: Why the newline?

Comment (ccqpein) [reply libraries.lisp:15]: Do you mean that line 15 and line 14 can merge together?

Comment (Slids) [reply libraries.lisp:15]: yes

Comment (ccqpein) [reply libraries.lisp]: No specific reason. Good catch. I just formatted it, how is it now?

Comment (Slids) [reply libraries.lisp]: lgtm

**Pull request merged**